feat(AVM)!: Enable tx=>execution lookup

sirasistant opened the pull request:
Enables the lookup now that it's passing bulk test, and reduces two columns in the tx trace for context ids

IlyasRidhuan commented on barretenberg/cpp/pil/vm2/tx.pil:
```diff
@@ -229,116 +219,112 @@ namespace tx;
+        prev_note_hash_tree_root,
+        prev_note_hash_tree_size,
+        prev_num_note_hashes_emitted,
+        prev_nullifier_tree_size,
```
are we missing the `prev_nullifier_tree_root`?

sirasistant committed:
feat(AVM)!: Enable tx=>execution lookup

Enables the lookup now that it's passing bulk test, and reduces two columns in the tx trace for context ids
